Automattic/wp-calypso
# Add missing 'Commisson fee ...' items to the /start/plans comparison grid

Related to Automattic/martech#2553

## Proposed Changes

In a recent update it was noticed (https://github.com/Automattic/martech/issues/2553#issuecomment-1894940665) that the two 'Commission fee ...' items are not displayed in the comparison grid on `/start/plans` or `/plans/:site`, while these are displayed on `/pricing`. This change adds the missing items.

## Testing Instructions

1. Visit `/start/plans` or `/plans/:site`
2. Scroll down and click 'Compare plans'
3. Scroll down the comparison grid and confirm that the two 'Commission fee ...' items are displayed

| Before | After |
| -- | -- |
|![CleanShot 2024-01-19 at 15 47 09@2x](https://github.com/Automattic/wp-calypso/assets/69198925/8d883433-f8af-4a13-9801-81102c329ec9)|![CleanShot 2024-01-19 at 15 45 57@2x](https://github.com/Automattic/wp-calypso/assets/69198925/ff0bce89-d3fa-461f-9f87-f44d03fc9e88)|



## Pre-merge Checklist
